packages/ui-transfer-components/lib/components/consent-request.component.ts

```diff
@@ -47,14 +22,24 @@ export class ConsentRequestComponent extends LitElement {
 
   render(): TemplateResult {
 
+    const hiddenPredicates = [
+      'https://schema.org/uri',
+      'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',
+      'http://www.w3.org/2006/vcard/ns#value',
+    ];
+
+    const predicatesToShow = this.purpose.predicates.filter((pred) => !hiddenPredicates.includes(pred));
+    const translatedPredicates = predicatesToShow.map((pred) => predicateTranslations[pred] ?? pred);
+    const uniqueTranslatedPredicates = translatedPredicates.filter((val, index, self) => self.indexOf(val) === index);
```

Comment:
This should also work and is cleaner. Could you test this?

```suggestion
    const uniqueTranslatedPredicates = Array.from(new Set(translatedPredicates));
```